src/main/resources/swagger/api-docs.yaml

```
@@ -9012,6 +9059,9 @@ components:
           enum:
             - NoNewCalls
             - ContinueWhilePossible
+        userComment:
+          type: string
+          description: Comment for the submission
```

Maybe worth matching the `description` of user comment below?
```suggestion
          description: Freeform user defined description, optional.
```